packages/tools/fluid-runner/src/fluidRunner.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as yargs from "yargs";
+import { exportFile } from "./exportFile";
+
+function fluidRunner() {
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    yargs
+        .strict()
+        .version(false)
+        .command(
+            "exportFile",
+            "Generate an output for a local snapshot",
+            // eslint-disable-next-line @typescript-eslint/no-shadow
+            (yargs) =>
+                yargs
+                    .option("codeLoader", {
+                        describe: "Name of the code loader bundle",
+                        type: "string",
+                        demandOption: true,
+                    })
+                    .option("inputFile", {
+                        describe: "Name of the file containing local ODSP snapshot",
+                        type: "string",
+                        demandOption: true,
+                    })
+                    .option("outputFolder", {
+                        describe: "Name of the output file",
+                        type: "string",
+                        demandOption: true,
+                    })
+                    .option("scenario", {
```

Comment:
At minimum, we need to pass this argument to createContainerAndExecute().
It would be better if it was more generic (like all the unknown args are passed to the createContainerAndExecute), such that solution is more versatile.